examples/AN00162_i2s_loopback_demo/Makefile

```diff
@@ -1,7 +1,14 @@
 # The TARGET variable determines what target system the application is
 # compiled for. It either refers to an XN file in the source directories
 # or a valid argument for the --target option when compiling
-TARGET = XR-AUDIO-216-MC
+XCOREAI ?= 0
+ifeq ($(XCOREAI),0)
```

Comment:
Assumes that not XCOREAI means XCORE-200. Should check for XCOREAI == 1 and have the XCORE-AI statements in the initial clause with the XCORE-200 statements in the else clause.  Please fix now or raise an issue for v4.5.0.